### ISSUE TITLE
Why do you change the maxpool layer of resnet?

### ISSUE THREAD

**Issue by caryleo** [ruotianluo/self-critical.pytorch#77]
Hi, dear @ruotianluo , thanks for your code.
I'm trying to read the code and I find your change in the resnet.py:
`self.maxpool = nn.MaxPool2d(kernel_size=3, stride=2, padding=0, ceil_mode=True)` 
I compare your version with the offical one, you change to cancel the padding and use ceil rather than floor.
Could you please explain this change? I can't figure out why and what the influence is. Thank you. :)
Best Regards.

**Comment by ruotianluo** [comment]
This is because I am using caffe resnet pretrained model, and that's what caffe does.

**Comment by caryleo** [comment]
I see, thank you :)
